### DISCUSSION TITLE
Icon for Blog in navigation

### DISCUSSION THREAD

**Discussion by holgerflick** [squidfunk/mkdocs-material#6599] [Q&A]
The blog plugin automatically inserts itself into the navigation or it can be mapped in the `mkdocs.yml` file. However, the `index.md` file in the blog directory does not support `icon:`. I also tried with individual posts.

Is it possible to assign an icon to the `Blog` item in the navigation? If not, I will file a suggestion.

**Comment by kamilkrzyskow** [comment]
Hello @holgerflick,
I don't see this behaviour with my example:
[icons-home-for-holgerflick.zip](https://github.com/squidfunk/mkdocs-material/files/13836294/icons-home-for-holgerflick.zip)
it's based on your other question: https://github.com/squidfunk/mkdocs-material/discussions/6600
It uses 2 `blog` plugin instances, and every top navigation item has an icon. So if you're still facing the problem, please provide more info.
